docs/source/additional/faq.rst

```diff
@@ -27,4 +27,15 @@ Although Substra has been designed to work especially well in healthcare setting
 
 How can I be sure Substra is secure enough to be used with my private data?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Substra is regularly audited with rigorous security standards (both code source audit and penetration tests). On top of that, by design, private data is never shared between different organizations. The software also provides full traceability on which algorithms were used on each dataset.
\ No newline at end of file
+Substra is regularly audited with rigorous security standards (both code source audit and penetration tests). On top of that, by design, private data is never shared between different organizations. The software also provides full traceability on which algorithms were used on each dataset.
+
+What is the roadmap for Substra?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The roadmap for Substra is primarily decided by engineers at Owkin. These decisions are based on requirements from active and potential FL projects that may or may not involve Owkin. Based on our expections, please find a list of features below that plan to focus on in the near future. Please know however that this is not a very strict roadmap and the direction of the project can alter at any moment.
+
+* **More features for Federated Analytics:** The Substra library does support FA currently but one of our goals is make this more user friendly and easily accessible.
+* **Introduce more FL Strategies:** Substra aims to be a complete FL framework and one way we hope to facilitate FL projects is by adding more strategies. We hope that by implementing these strategies within the library, we can encourage more experimentation by data scientists. We would also be interested in allowing users to define their own FL strategies.
+* **Security Improvements:** Although Substra has already been proven to be quite secure, any vulnerabilities found would be a priority to work on, and we would continue to add more features for traceability and security.
+
```

Comment:
I would add something on the merge of Substra and SubstraFL (fix some redundants concepts and have one unified library) + the data concept revamp to simply the data concepts (dataset, datasample, train_data_node, opener, etc.)